### PR TITLE
Provision vic-ui in the ova

### PR DESCRIPTION
**Pull request by morris-jason**
VIC Appliance Checklist:
- [x] Up to date with `master` branch
- [ ] Added tests
- [ ] Considered impact to upgrade
- [ ] Tests passing
- [ ] Updated documentation
- [ ] Impact assessment checklist

If this is a feature or change to existing functionality, consider areas of impact with the [Impact
Assessment Checklist](https://github.com/vmware/vic-product/blob/master/installer/docs/CHANGE.md)

Fixes vmware/vic-product#1433.
- Downloads vic-ui plugin artifacts as a component of the ova.
- Updates the fileserver provisioner to package vic-ui with the correct version numbers.
- Moves the plugin install scripts from vic-ui to vic-product.

### CODE REVIEW

**Comment by mdharamadas1** [comment]
@morris-jason Does the tests added here work? If not, we have to make sure they are excluded from nightly runs until they are fixed. It can be done using tag `skip`.

**Comment by morris-jason** [comment]
@mdharamadas1 they will not work until https://github.com/vmware/vic-product/issues/1740 is complete. This pr is to our feature branch, so can we fix the tests during https://github.com/vmware/vic-product/issues/1701?

**Comment by morris-jason** [comment]
@mdharamadas1 they will not work until https://github.com/vmware/vic-product/issues/1740 is complete. This pr is to our feature branch, so can we fix the tests during https://github.com/vmware/vic-product/issues/1701?

File: tests/manual-test-cases/TestGroups.md [hunk 1]
```diff
@@ -10,3 +10,4 @@ VIC Integration Test Suite
 -
 [Group 7 - Upgrade](Group7-Upgrade/TestCases.md)
 -
+[Group 18 - VIC UI](Group18-VIC-UI/TestCases.md) TODO(morris-jason)
```
[the remark this answers is not in the document]
Comment (morris-jason) [reply tests/manual-test-cases/TestGroups.md]: https://github.com/vmware/vic-product/issues/1771

**Pull request merged**